bin/juxtaposer/README.md

```diff
@@ -44,6 +44,180 @@ Note: More comparison types may be added in the future.
 
 ---
 
+## Running the code
+
+You can run and test this code on your machine easily with a few things:
+- [Docker](https://docker.com)
+- [Golang](https://golang.org/dl/) 1.12 or higher
+- [Secretless-broker](https://github.com/cyberark/secretless-broker) (optional)
+- A writable folder for shared socket files (e.g. `/sock` in these configurations)
```

Comment:
Is this optional too?  What if you were only testing `tcp` backends?

Reply:
It's optional too - I'll fix in followup commit